apm-agent-plugins/apm-jms-plugin/src/main/java/co/elastic/apm/agent/jms/JmsMessageListenerInstrumentation.java

```diff
@@ -63,14 +63,15 @@ public JmsMessageListenerInstrumentation(ElasticApmTracer tracer) {
 
     @Override
     public ElementMatcher<? super TypeDescription> getTypeMatcher() {
-        return not(isInterface()).and(hasSuperType(named("javax.jms.MessageListener")));
+        return not(isInterface())
```

Comment:
This formatting may make it a bit clearer than the unnecessary parenthesis? YMMV

```
return not(isInterface())
    .and(
        hasSuperType(named("javax.jms.MessageListener"))
            .or(hasSuperType(named("org.springframework.jms.listener.SessionAwareMessageListener")))
    );
```

Reply:
Yes, there is an extra unnecessary pair